src/components/Table/Table.react.js

```diff
@@ -26,17 +26,20 @@ export class Table extends React.Component {
       columnHeadings: this.context.tableStore.getColumns(),
       rows: this.context.tableStore.getRows(),
       currentPage: this.context.tableStore.getCurrentPage(),
-      totalPages: this.context.tableStore.getTotalPages()
+      totalPages: this.context.tableStore.getTotalPages(),
+      tableCaption: this.context.tableStore.getTableCaption()
     };
   }
 
   render() {
     var headings = this.state.columnHeadings;
+    var tableCaption = this.state.tableCaption;
 
     return (
       <div className="panel panel-responsive">
         <div className="table-responsive">
           <table className="table table-hover table-striped">
+            <caption>{tableCaption}</caption>
```

Comment:
I think it does. ATM the margins are equal
![image](https://cloud.githubusercontent.com/assets/2174682/13655458/9607f27a-e6b3-11e5-8211-368970d8b323.png)